src/CMakeLists.txt

```diff
@@ -18,7 +18,12 @@ file(GLOB toit_core_SRC
   "*.cc"
   )
 
-list(FILTER toit_core_SRC EXCLUDE REGEX "/(toit|toit_run_image).cc$")
+list(FILTER toit_core_SRC EXCLUDE REGEX "/(toit|toit_run_image|vm).cc$")
```

Comment:
Add a quick comment to explain why we're dividing the source code up into parts here?